app/controllers/home_controller.rb

```diff
@@ -1,6 +1,6 @@
 # Home Controller
 class HomeController < ApplicationController
-  skip_before_filter :authenticate_user!
+  prepend_before_filter :require_no_authentication, only: [:cancel ]
```

Comment:
I found it on stackover flow as the employee were not getting added through login page i got an error so i used it.